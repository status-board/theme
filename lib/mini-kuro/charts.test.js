"use strict";

var _charts = _interopRequireDefault(require("./charts"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('Charts should match snapshot', function () {
  expect(_charts["default"]).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9taW5pLWt1cm8vY2hhcnRzLnRlc3QudHMiXSwibmFtZXMiOlsidGVzdCIsImV4cGVjdCIsImNoYXJ0cyIsInRvTWF0Y2hTbmFwc2hvdCJdLCJtYXBwaW5ncyI6Ijs7QUFBQTs7OztBQUVBQSxJQUFJLENBQUMsOEJBQUQsRUFBaUMsWUFBWTtBQUM3Q0MsRUFBQUEsTUFBTSxDQUFDQyxrQkFBRCxDQUFOLENBQWVDLGVBQWY7QUFDSCxDQUZHLENBQUoiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgY2hhcnRzIGZyb20gJy4vY2hhcnRzJztcblxudGVzdCgnQ2hhcnRzIHNob3VsZCBtYXRjaCBzbmFwc2hvdCcsICgpOiB2b2lkID0+IHtcbiAgICBleHBlY3QoY2hhcnRzKS50b01hdGNoU25hcHNob3QoKTtcbn0pO1xuIl19