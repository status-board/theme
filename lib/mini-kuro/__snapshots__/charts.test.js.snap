// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Charts should match snapshot 1`] = `
Object {
  "axis": Object {
    "legendColor": "#fff",
    "textColor": "#fff",
    "tickColor": "#fff",
  },
  "colors": Array [
    "#66c2a5",
    "#fc8d62",
    "#8da0cb",
    "#e78ac3",
    "#a6d854",
  ],
  "grid": Object {
    "stroke": "#222",
  },
  "tooltip": Object {
    "background": "#000",
    "border": "1px solid #fff",
    "borderRadius": 0,
    "boxShadow": "none",
    "color": "#fff",
    "fontSize": "1.4vmin",
  },
}
`;
